ios/Classes/Codables.swift

```diff
@@ -242,7 +242,7 @@ class PolarPpgSampleCodable: Encodable {
 }
 
 typealias PolarPpiSample = (
-  hr: Int, ppInMs: UInt16, ppErrorEstimate: UInt16, blockerBit: Int, skinContactStatus: Int,
+    timeStamp: UInt64, hr: Int, ppInMs: UInt16, ppErrorEstimate: UInt16, blockerBit: Int, skinContactStatus: Int,
```

Comment:
This field need added to `PolarPpiSample` in Dart
It also needs handled in the `encode` function